modules/default/weather/hourly.njk

```diff
@@ -21,15 +21,27 @@
                         {% endif %}
                     </td>
                 {% endif %}
+                {% if config.showHumidity %}
```

Comment:
please add the new config options to the defaults object at the top of the file

Reply:
Do you mean the defaults in the weather.js?

Reply:
yes, it is a new config property, so it needs to havea default value set, and the defaults:{} object contains those 
as documented in the developers guide for modules